GDJS/Runtime/runtimebehavior.js

```diff
@@ -4,13 +4,19 @@
  * This project is released under the MIT License.
  */
 
+/**
+ * @typedef behaviorData
+ * @property {string} name The name of the behavior (for getting from an object (object.getBehavior) for example)
```

Comment:
"for getting from an object" does not sounds very clear.
Rather:
The name of the behavior. You can query a behavior using its name in a gdjs.RuntimeObject using `getBehavior` method.

Reply:
Should I also add links (`The name of the behavior. You can query a behavior using its name in a {@link gdjs.RuntimeObject} using {@link gdjs.RuntimeObject.prototype.getBehavior} method.`) ?